tests/prefetch_related/tests.py

```diff
@@ -704,6 +716,30 @@ def test_nested_prefetch_related_are_not_overwritten(self):
             self.room2_1
         )
 
+    def test_apply_rel_filters_deprecation_shim(self):
+        # Simulate a missing `_apply_rel_filters` method.
```

Comment:
some names have backticks, in these comments and some don't. Personally I would just use _apply_rel_filters() but as long as you're consistent I don't mind either way.